core/templates/dev/head/pages/exploration-editor-page/feedback-tab/services/thread-data.service.spec.ts

```diff
@@ -129,15 +129,15 @@ describe('retrieving threads service', function() {
       suggestions: mockGeneralSuggestionThreads
     });
 
-    ThreadDataService.fetchThreads(function() {
+    ThreadDataService.fetchThreads().then(threadData => {
```

Comment:
All this stuff is async, so shouldn't there be some use of done() here? See last bullet point in https://github.com/oppia/oppia/wiki/Frontend-test-best-practices#best-practices-for-good-tests

Reply:
Done.